isis/src/qisis/apps/qnet/qnet.xml

```diff
@@ -30,6 +30,10 @@
     </p>
 
     <p>
+      This program should be started by typing its name in a terminal, with no arguments. 
+      This will start blank versions of the main Qnet window and the control network 
+      navigator window.
+      

```

Comment:
Probably better to word in such a way that makes it obvious this is a GUI application. As ISIS GUI apps follow the same paradigm. 

So, start with something like: "This is an ISIS GUI application. To run it with entering the command in the terminal with no parameters to start a blank session and select your files through the navigator windows."

Reply:
I added an update to the text ([d4153ee](https://github.com/DOI-USGS/ISIS3/pull/5373/commits/d4153ee19fc5f824565775cd8b52cd422c9bb3fc))